examples/config-metrics.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.1.0-dev",
+    "storage": {
+        "rootDirectory": "/tmp/zot"
+    },
+    "http": {
+        "address": "127.0.0.1",
+        "port": "8080"
+    },
+    "log": {
+        "level": "debug"
+    },
+    "extensions": {
+        "metrics": {
```

Comment:
If we are including metrics feature in minimal binary than we should use metrics section on top level

Reply:
I think there are basic in-memory metrics which are available.
"metrics" under extensions are the full prometheus implementation.
This is fine actually.